cli_li3ds/import_orimatis.py

```diff
@@ -283,7 +283,7 @@ def datasource_image(session, referential, datasource,
     if image_file_ext:
         image_path = image_path.with_suffix(image_file_ext)
     image_path = os.path.normpath(str(image_path))
-    uri = 'file:{}'.format(image_path)
+    uri = 'file://{}'.format(image_path)
```

Comment:
See https://github.com/LI3DS/pg-li3ds/blob/master/extension/li3ds--1.0.0.sql#L79-L121 for the corresponding check function.